### PR TITLE
Update core sdk, fix create user and update user controllers

### PR DESCRIPTION
**Pull request by ejiro-wilkie**

### CODE REVIEW

File: Humin-Man.Services/UserService.cs [hunk 1]
```diff
@@ -120,15 +120,21 @@ public async Task UpdateAsync(UpdateUserInputModel input)
...
-            };
+                // Handle the error, e.g., throw an exception or return an error response
+            }
+
+            user.FirstName = input.FirstName;
+            user.LastName = input.LastName;
+            user.UserName = input.UserName;
+            user.Email = input.Email;
+            user.PhoneNumber = input.PhoneNumber;
+            user.SecurityStamp = "";
```
Comment (moh-nabih) [inline Humin-Man.Services/UserService.cs:136]: remove this line

File: Humin-Man/Startup.cs [hunk 2]
```diff
@@ -61,7 +61,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
-                app.UseDatabaseErrorPage();
+                app.UseMigrationsEndPoint();
```
Comment (moh-nabih) [inline Humin-Man/Startup.cs:64]: Why did you change this?

Comment (ejiro-wilkie) [reply Humin-Man/Startup.cs:64]: i was getting an error and found out that `app.UseDatabaseErrorPage()` is obsolete 
https://github.com/aspnet/Announcements/issues/432
